FreeIT/src/lesson_10/Application.java

```diff
@@ -0,0 +1,237 @@
+package lesson_10;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.InputMismatchException;
+import java.util.List;
+import java.util.ListIterator;
+import java.util.Scanner;
+
+public class Application {
+
+	private Library libr;
+
+	public Application() {
+		super();
```

Comment:
Why do you need constructor of super here?

Reply:
Shouldn't this be by the code convention?

Reply:
No )